### PR TITLE
Mz/w 14720788/diff refresh to sf

### PR DESCRIPTION
**Pull request by mingxuanzhangsfdx**
### What does this PR do?
Change the style of diff files, folders & refresh sobject to sf

### What issues does this PR fix or reference?
@W-14720788@

### Functionality Before
sfdx style

### Functionality After
sf style

### CODE REVIEW

File: packages/salesforcedx-vscode-core/src/commands/sourceDiff.ts [hunk 1]
```diff
@@ -21,15 +21,15 @@ import { FilePathGatherer, SfdxCommandlet, SfdxWorkspaceChecker } from './util';
...
   if (!sourceUri) {
     const editor = vscode.window.activeTextEditor;
     if (editor && editor.document.languageId !== 'forcesourcemanifest') {
       sourceUri = editor.document.uri;
     } else {
-      const errorMessage = nls.localize('force_source_diff_unsupported_type');
+      const errorMessage = nls.localize('source_diff_unsupported_type');
       telemetryService.sendException('unsupported_type_on_diff', errorMessage);
-      notificationService.showErrorMessage(errorMessage);
+      await notificationService.showErrorMessage(errorMessage);
```
Comment (CristiCanizales) [inline packages/salesforcedx-vscode-core/src/commands/sourceDiff.ts:32]: same here

File: packages/salesforcedx-vscode-core/src/commands/sourceDiff.ts [hunk 2]
```diff
@@ -38,13 +38,15 @@ export async function forceSourceDiff(sourceUri?: vscode.Uri) {
 
   const defaultUsernameorAlias = WorkspaceContext.getInstance().username;
   if (!defaultUsernameorAlias) {
-    notificationService.showErrorMessage(nls.localize('missing_default_org'));
+    await notificationService.showErrorMessage(
```
Comment (CristiCanizales) [inline packages/salesforcedx-vscode-core/src/commands/sourceDiff.ts:41]: and here

File: packages/salesforcedx-vscode-core/src/commands/sourceDiff.ts [hunk 3]
```diff
@@ -53,17 +55,17 @@ export async function forceSourceDiff(sourceUri?: vscode.Uri) {
...
   if (!explorerPath) {
     const editor = vscode.window.activeTextEditor;
     if (editor && editor.document.languageId !== 'forcesourcemanifest') {
       explorerPath = editor.document.uri;
     } else {
-      const errorMessage = nls.localize('force_source_diff_unsupported_type');
+      const errorMessage = nls.localize('source_diff_unsupported_type');
       telemetryService.sendException('unsupported_type_on_diff', errorMessage);
-      notificationService.showErrorMessage(errorMessage);
+      await notificationService.showErrorMessage(errorMessage);
```
Comment (CristiCanizales) [inline packages/salesforcedx-vscode-core/src/commands/sourceDiff.ts:68]: and here

File: packages/salesforcedx-vscode-core/src/commands/sourceDiff.ts [hunk 4]
```diff
@@ -72,7 +74,9 @@ export async function forceSourceFolderDiff(explorerPath: vscode.Uri) {
 
   const username = WorkspaceContext.getInstance().username;
   if (!username) {
-    notificationService.showErrorMessage(nls.localize('missing_default_org'));
+    await notificationService.showErrorMessage(
```
Comment (CristiCanizales) [inline packages/salesforcedx-vscode-core/src/commands/sourceDiff.ts:77]: and here (?

**Review by CristiCanizales — COMMENTED** [review]
Just one comment :)

**Comment by CristiCanizales** [comment]
QE notes:
- Three commands continue working as expected ✅ 
- refresh sobjects on startup ✅ 
- recreates sobjects folder if it's not there ✅

File: packages/salesforcedx-vscode-core/src/conflict/directoryDiffer.ts [hunk 5]
```diff
@@ -163,19 +163,19 @@ export async function diffOneFile(
...
-            'force_source_diff_title',
+            'source_diff_title',
             defaultUsernameorAlias,
             filePart,
             filePart
           )
         );
       } catch (err) {
-        notificationService.showErrorMessage(err.message);
+        await notificationService.showErrorMessage(err.message);
```
[the remark this answers is not in the document]
Comment (mingxuanzhangsfdx) [reply packages/salesforcedx-vscode-core/src/conflict/directoryDiffer.ts:173]: [From what I learnt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await), the return type Thenable also needs await. My vscode lint check also reminds me to do so.

**Review by CristiCanizales — APPROVED** [review]
Thanks for helping with this!
:shipit:

**Pull request merged**